README.md

```diff
@@ -1,26 +1,341 @@
+The template link must be in Go tpl format and typically will include the {{.To.Original}} variable that will be replaced by the last bumped version.
+
+Example dictionary file:
+```yaml
+dictionary:
+  a-dependency: "https://github.com/my-org/a-dependency/releases/tag/{{.To.Original}}"
+```
+
+`changelog.yaml` before linking dependencies:
+```yaml
+dependencies:
+  - name: a-dependency
+    from: "2.1.0"
+    to: "3.0.0"
+```
+
+`changelog.yaml` after linking dependencies:
+```yaml
+dependencies:
+  - name: a-dependency
+    from: "2.1.0"
+    to: "3.0.0"
+    meta:
+      changelog: https://github.com/my-org/a-dependency/releases/tag/3.0.0
+```
 
-## Install
+### Automated releasing
 
-RT can be installed with Go by executing the following command:
+The ultimate goal of the release toolkit is to allow for fully automated release pipelines that produce meaningful changelogs and respect semver standards. For this reason, the recommended way of implementing the toolkit is as a scheduled job that runs on the `main`/`master` branch of your source tree, for example every tuesday at midnight.
 
+The release toolkit allows you to build a pipeline that creates a release with its changelog and version automatically if there are any unreleased changes in your main branch.
+
+However, it is possible that, in some cases, you may want to hold changes from being released as you want to "pack" them with future changes for UX reasons. To this end, the release toolkit includes the concept of holding releases. A release can be held by including a `## Held` L2 header in `CHANGELOG.md`. This header must be followed by a paragraph explaining why automated releases are on hold:
+
+```markdown
+# Changelog
+This is based on blah blah blah
+
+## Held
+
+Holding release as it contains massive breaking changes
+
+## Unreleased
+
+### Breaking
+- Support has been removed
+
+## v1.2.3 - 20YY-DD-MM
+```
+
+The presence of this header does not modify the fundamental behavior of any command. It does, however, cause `generate-changelog` to include a `held: true` property in `changelog.yaml`:
+
+```yaml
+held: true
+changes:
+  - type: breaking
+    message: Support has been removed
+```
+
+This flag can be readily checked using the `is-held` action/command. `is-held` will echo the value of this property to `stdout`, and can be configured to exit with non-zero if `held` is set to `true`. While running as an action, it will also set the `is-held` step output to the value of the property. This allows your pipeline to react to this property and skip the release process if a `## Held` entry is present in your `CHANGELOG.md`.
+
+`update-changelog` will not behave any different in presence of a `## Held` header, and will remove it before integrating the changelog for the latest release.
+
+### Full example of automated releasing pipeline
+
+![release-toolkit.png](release-toolkit.png)
```

Comment:
Will this hyperlink work on the main page of the repo? I don't think it will.

Reply:
It seems to be working, at least, when the file is _pre-viewed_ 

<img width="1052" alt="image" src="https://user-images.githubusercontent.com/442627/196692619-d1806a39-1126-4665-84e6-081297bf3e8b.png">

Reply:
I remember that for a long time, images in `README.md` wouldn't work when the readme is viewed from the repo root (`https://github.com/newrelic/release-toolkit`) because they would generate an image source like:
`https://github.com/newrelic/release-toolkit/release-toolkit.png`
Which Github does not handle well.

Maybe Github has fixed this 🤔

Reply:
Yes it does work.